ragna/source_storages/_lancedb.py

```diff
@@ -32,84 +39,81 @@ def __init__(self) -> None:
         super().__init__()
 
         import lancedb
-        import pyarrow as pa
+
+        self._tokens = 0
+        self._embeddings = 0
 
         self._db = lancedb.connect(ragna.local_root() / "lancedb")
-        self._schema = pa.schema(
+
+    _VECTOR_COLUMN_NAME = "embedded_text"
+
+    def store(
+        self,
+        documents: list[Embedding],
+        *,
+        chat_id: uuid.UUID,
+    ) -> None:
+        import pyarrow as pa
+
+        _schema = pa.schema(
             [
                 pa.field("id", pa.string()),
                 pa.field("document_id", pa.string()),
                 pa.field("page_numbers", pa.string()),
                 pa.field("text", pa.string()),
                 pa.field(
                     self._VECTOR_COLUMN_NAME,
-                    pa.list_(pa.float32(), self._embedding_dimensions),
+                    pa.list_(pa.float32(), len(documents[0].embedding)),
```

Comment:
Since that might not be super obvious at first glance, can we create a variable for this above the block

```
dimensionality = len(documents[0].embedding))
_schema = ...
```